<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiClass</class>
 <widget class="QMainWindow" name="QtGuiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>QtGui</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item alignment="Qt::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_simulation">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Setup Simulation</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_optimization">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Setup/Start Optimization</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_force_opti_end">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Force Optimization end</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_start_learning">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Start Learning</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_start">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="sliderPosition">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string notr="true">Speed : 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pb_simulation</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiClass</receiver>
   <slot>setup_simulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_optimization</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiClass</receiver>
   <slot>setup_optimization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_start_learning</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiClass</receiver>
   <slot>start_learning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_force_opti_end</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiClass</receiver>
   <slot>stop_opti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtGuiClass</receiver>
   <slot>set_speed_value(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtGuiClass</receiver>
   <slot>set_speed_value(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setup_simulation()</slot>
  <slot>setup_optimization()</slot>
  <slot>start_learning()</slot>
  <slot>stop_opti()</slot>
  <slot>set_speed_value(int)</slot>
 </slots>
</ui>
